src/v0/destinations/azure_event_hub/transform.js

```diff
@@ -5,20 +5,6 @@ function process(event) {
   const { message, destination } = event;
   const integrationsObj = getIntegrationsObj(message, "azure_event_hub");
   const topic = integrationsObj?.topic || destination.Config?.topic;
-  // TODO: Remove commented lines after server release
```

Comment:
Please revert this change but refactor the code.

Reply:
done